articles/install-guide/index.md

```diff
@@ -1,53 +1,223 @@
-> [!div class="button"]
-> [Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=quantum.quantum-devkit-vscode)
+    ```bash
+    pip install qsharp
+    ```
 
-Then, check our [quickstart guide](xref:microsoft.quantum.write-program?tabs=tabid-vscode) for step-by-step instructions for your first quantum program.
+1. Verify the installation by creating a `Hello World` application
 
+    - Create a minimal Q# operation, by creating a file called `Operation.qs`, and adding the following code to it:
 
-### Python ###
+        ```qsharp
+        namespace HelloWorld
+        {
+            open Microsoft.Quantum.Intrinsic;
+            open Microsoft.Quantum.Canon;
 
-If you are a Python user, check out our [getting started with Python and the Quantum Development Kit guide](xref:microsoft.quantum.install.python).
+            operation SayHello() : Result {
+                Message("Hello from quantum world!");
+                return Zero;
+            }
+        }
+        ```
 
+    - Create a Python program called `hello_world.py` to call the Q# `SayHello()` operation:
 
-## Learn Quantum Computing with Q# ##
+        ```python
+        import qsharp
 
-* [Quantum Katas](https://github.com/Microsoft/QuantumKatas)
-* [Samples](https://github.com/Microsoft/Quantum)
-* [Quantum Computing Concepts](xref:microsoft.quantum.concepts.intro)
-* [Microsoft Quantum Blog](https://cloudblogs.microsoft.com/quantum/?ext)
-* [Q# Blog](https://devblogs.microsoft.com/qsharp/)
+        from HelloWorld import SayHello
 
+        SayHello.simulate()
+        ```
 
+    - Run the program:
+
+        ```bash
+        python hello_world.py
+        ```
+
+    - Verify the output. Your program should output the following lines:
+
+        ```bash
+        Hello from quantum world!
+       0
+       ```
+
+## Develop with Jupyter notebooks
+
+1. Pre-requisites
+
+    - [Python](https://www.python.org/downloads/) 3.6 or later
+    - [Jupyter Notebook](https://jupyter.readthedocs.io/en/latest/install.html)
+    - [.NET Core SDK 2.1 or later](https://www.microsoft.com/net/download)
+
+1. Install the `iqsharp` package
+
+    ```bash
+    dotnet tool install -g Microsoft.Quantum.IQSharp
+    dotnet iqsharp install
+    ```
+
+1. Verify the installation by creating a `Hello World` application
+
+    - Run the following command to start the notebook server:
+
+        ```bash
+        jupyter notebook
+        ```
+
+    - Browse to the URL shown on the command line. For example: [http://localhost:8888/?token=c790a52ba54f0cf77465c3c8983d776348285b0280d91b85]
+
+    - Create a Jupyter notebook with a Q# kernel, and add the following code to the first notebook cell:
+
+        ```qsharp
+        operation SayHello () : Unit {
+            Message("Hello from quantum world!");
+        }
+        ```
+
+    - Run this cell of the notebook:
+
```

Comment:
It looks like the actual cell is missing here?

![image](https://user-images.githubusercontent.com/31516/65790817-40d87080-e115-11e9-94c8-323c9f015ce9.png)

Reply:
I didn't intend the cell be there but I agree it is not clear. I've added a screenshot